x/hub-genesis/types/state.go

```diff
@@ -1,9 +1,19 @@
 package types
 
-// validate state
+import (
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
 func (s State) Validate() error {
-	if !s.GenesisTokens.IsValid() {
-		return ErrInvalidGenesisTokens
+	for _, a := range s.GetGenesisAccounts() {
+		if err := a.GetAmount().Validate(); err != nil {
```

Comment:
validate denom is the native denom?

Reply:
discuss here https://github.com/dymensionxyz/dymension-rdk/pull/449#discussion_r1647542987